internal/tools/inspect/manifest.go

```diff
@@ -105,7 +105,7 @@ func gotoEntry(r *dwarf.Reader, tag dwarf.Tag, name string) bool {
 func findEntry(r *dwarf.Reader, tag dwarf.Tag, name string) (*dwarf.Entry, error) {
 	for {
 		entry, err := r.Next()
-		if err == io.EOF || entry == nil {
+		if errors.Is(err, io.EOF) || entry == nil {
```

Comment:
I do not understand why we are even checking for `io.EOF`. See https://pkg.go.dev/debug/dwarf#Reader

```suggestion
		if entry == nil {
```

Reply:
Yeah, good point. Now that I see it again, I made the same conclusion when I looked at this a few months ago.